### ISSUE TITLE
The header of the details' column should have a label

### ISSUE THREAD

**Issue by NDBman** [i-Cell-Mobilsoft-Open-Source/icell-data-table#167]
**The problem**
In terms of a11y when you navigate through the names of the column headers the first cell, which would name the details column, won't have any labels and that can be confusing for the visually impaired.

**Desired solution**
There should be a way to define a label for that column.

**Alternatives**
- A default translate key would also be enough with the possibility to show and hide it.
- An aria-label which can be read by the screen reader. Drawback: this would violate inclusive design principles.

**Attachment**
![Képernyőfotó 2021-05-31 - 17 02 21](https://user-images.githubusercontent.com/11921146/120212352-29c71b80-c232-11eb-8bd4-1865ca6e25a2.png)
